lib/CModHandler.cpp

```diff
@@ -352,7 +352,7 @@ void CIdentifierStorage::finalize()
 		}
 		logMod->error("All known identifiers were dumped into log file");
 	}
-	assert(errorsFound == false);
+	

```

Comment:
Why change?

Reply:
failed here if some objects are missing on original data

Reply:
Then fix?
This is almost guaranteed to result in crash down the line. What the point of broken demo support?

Reply:
nope, game operates fine. The problem is that if you have some null identifiers, it gives such issue.
e.g try to have line in mod and you will have same error
`"core:vialOfDragonBlood": null,`

Reply:
What exactly lead to this error?
Normally, this is due to unfulfilled identifier request, meaning that somewhere in code instead of writing vial of dragonblood ID we kept uninitialized field, that will lead to crash when it is accessed. Or is this not the case here?